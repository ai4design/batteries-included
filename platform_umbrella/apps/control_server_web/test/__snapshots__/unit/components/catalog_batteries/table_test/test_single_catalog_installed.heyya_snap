<div class="overflow-y-auto px-4 sm:overflow-visible sm:px-0">
  <table class="w-[40rem] mt-4 sm:w-full">
    <thead class="text-sm text-left leading-6 text-gray-darker dark:text-gray">
      <tr>
        <th class="p-0 pb-4 font-normal">Name</th><th class="p-0 pb-4 font-normal"></th><th class="p-0 pb-4 font-normal">Type</th><th class="p-0 pb-4 font-normal">Status</th><th class="p-0 pb-4 font-normal">Install</th>
        
      </tr>
    </thead>
    <tbody id="batteries-table" class="relative dark:divide-y dark:divide-gray-0 border-t border-gray-lighter dark:border-gray-darkest text-sm leading-6 text-gray-darkest dark:text-gray-lighter">
      <tr class="group ">
        <td class="p-0 px-2 ">
          <div class="block py-4">
            <span class="font-semibold text-gray-darkest dark:text-gray-lightest">
              
    Redis
  
            </span>
          </div>
        </td><td class="p-0 px-2 ">
          <div class="block py-4">
            <span class="">
              
    <div class="cursor-pointer " id="battery-redis-desc">
  <svg xmlns="http://www.w3.org/2000/svg" class="w-6 h-auto" aria-hidden="true" fill="none" viewBox="0 0 24 24" stroke="currentColor" stroke-width="1.5">
  <path stroke-linecap="round" stroke-linejoin="round" d="M9.879 7.519c1.171-1.025 3.071-1.025 4.242 0 1.172 1.025 1.172 2.687 0 3.712-.203.179-.43.326-.67.442-.745.361-1.45.999-1.45 1.827v.75M21 12a9 9 0 11-18 0 9 9 0 0118 0zm-9 5.25h.008v.008H12v-.008z"/>
</svg>
  <div class="hidden" id="tooltip-battery-redis-desc" phx-hook="Tooltip" data-target="battery-redis-desc" data-tippy-options="{}">
  <div>
    
    
      Redis is an in-memory data structure store, used as a database, cache, message broker, and streaming engine.
    
  
  </div>
</div>
</div>
  
            </span>
          </div>
        </td><td class="p-0 px-2 ">
          <div class="block py-4">
            <span class="">
              
    data
  
            </span>
          </div>
        </td><td class="p-0 px-2 ">
          <div class="block py-4">
            <span class="">
              
    <div class="flex text-success font-semi-bold">
  <div class="flex-initial">
    Active
  </div>
  <div class="flex-none ml-2">
    <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" aria-hidden="true" fill="none" viewBox="0 0 24 24" stroke="currentColor" stroke-width="1.5">
  <path stroke-linecap="round" stroke-linejoin="round" d="M9 12.75L11.25 15 15 9.75M21 12a9 9 0 11-18 0 9 9 0 0118 0z"/>
</svg>
  </div>
</div>
  
            </span>
          </div>
        </td><td class="p-0 px-2 ">
          <div class="block py-4">
            <span class="">
              
    
  
            </span>
          </div>
        </td>
        <td class="w-14 p-0">
          <div class="gap-4 lg:gap-6 flex whitespace-nowrap text-sm font-medium justify-around">
  
            
          
</div>
        </td>
      </tr>
    </tbody>
  </table>
</div>