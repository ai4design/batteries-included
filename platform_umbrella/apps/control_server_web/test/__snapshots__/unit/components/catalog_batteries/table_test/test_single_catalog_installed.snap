"<div id=\"batteries-table\" class=\"overflow-y-auto px-4 sm:overflow-visible sm:px-0\">\n  <table class=\"mt-4 w-full\">\n    <thead class=\"text-left text-base leading-6 text-gray-700\">\n      <tr>\n        <th class=\"p-0 pb-4 pr-6 text-pink-500\">\n          Type\n        </th><th class=\"p-0 pb-4 pr-6 text-pink-500\">\n          \n        </th><th class=\"p-0 pb-4 pr-6 text-pink-500\">\n          Group\n        </th><th class=\"p-0 pb-4 pr-6 text-pink-500\">\n          Status\n        </th>\n        <th class=\"relative p-0 pb-4\">\n          <span class=\"sr-only\">Actions</span>\n        </th>\n      </tr>\n    </thead>\n    <tbody class=\"relative divide-y divide-gray-100 border-t border-gray-200 text-sm leading-6 text-gray-700\">\n      <tr id=\"batteries-table-idx-0\" class=\"relative group hover:bg-gray-50\">\n        <td class=\"p-0 \">\n          <div>\n            <span class=\"absolute h-full w-4 top-0 -left-4 group-hover:bg-gray-50 sm:rounded-l-xl\"></span>\n            <span class=\"absolute h-full w-4 top-0 -right-4 group-hover:bg-gray-50 sm:rounded-r-xl\"></span>\n          </div>\n          <div class=\"block py-4 pr-6\">\n            <span class=\"relative font-semibold text-gray-900\">\n              \n    Redis\n  \n            </span>\n          </div>\n        </td><td class=\"p-0 \">\n          \n          <div class=\"block py-4 pr-6\">\n            <span class=\"relative \">\n              \n    <div class=\"cursor-pointer\" id=\"battery-redis-desc\">\n  <svg xmlns=\"http://www.w3.org/2000/svg\" aria-hidden=\"true\" class=\"w-6 h-auto\" fill=\"none\" stroke=\"currentColor\" stroke-width=\"1.5\" viewBox=\"0 0 24 24\">\n  <path stroke-linecap=\"round\" stroke-linejoin=\"round\" d=\"M9.879 7.519c1.171-1.025 3.071-1.025 4.242 0 1.172 1.025 1.172 2.687 0 3.712-.203.179-.43.326-.67.442-.745.361-1.45.999-1.45 1.827v.75M21 12a9 9 0 11-18 0 9 9 0 0118 0zm-9 5.25h.008v.008H12v-.008z\"/>\n</svg>\n  <div class=\"hidden\" id=\"tooltip-battery-redis-desc\" phx-hook=\"Tooltip\" data-target=\"battery-redis-desc\">\n  <div class=\"min-w-24 bg-white shadow-sm p-2 rounded-md\">\n    \n    \n      Redis is an in-memory data structure store, used as a database, cache, message broker, and streaming engine.\n    \n  \n  </div>\n</div>\n</div>\n  \n            </span>\n          </div>\n        </td><td class=\"p-0 \">\n          \n          <div class=\"block py-4 pr-6\">\n            <span class=\"relative \">\n              \n    data\n  \n            </span>\n          </div>\n        </td><td class=\"p-0 \">\n          \n          <div class=\"block py-4 pr-6\">\n            <span class=\"relative \">\n              \n    <div class=\"flex text-shamrock-500 font-semi-bold\">\n  <div class=\"flex-initial\">\n    Active\n  </div>\n  <div class=\"flex-none ml-2\">\n    <svg xmlns=\"http://www.w3.org/2000/svg\" aria-hidden=\"true\" class=\"h-6 w-6\" fill=\"none\" stroke=\"currentColor\" stroke-width=\"1.5\" viewBox=\"0 0 24 24\">\n  <path stroke-linecap=\"round\" stroke-linejoin=\"round\" d=\"M9 12.75L11.25 15 15 9.75M21 12a9 9 0 11-18 0 9 9 0 0118 0z\"/>\n</svg>\n  </div>\n</div>\n  \n            </span>\n          </div>\n        </td>\n        <td class=\"p-0 w-14\">\n          <div class=\"relative whitespace-nowrap py-4 text-right text-sm font-medium\">\n            <span class=\"relative ml-4 font-semibold leading-6 text-gray-900 hover:text-gray-700\">\n              \n    \n  \n            </span>\n          </div>\n        </td>\n      </tr>\n    </tbody>\n  </table>\n</div>"